Dockerfile

```diff
@@ -8,18 +8,27 @@ ARG DEBIAN_FRONTEND=noninteractive
 ENV PATH=/opt/conda/bin:$PATH
 ENV PYTHON_VERSION=${PYTHON_VER}
 
-COPY --from=condaforge/mambaforge:22.9.0-2 /opt/conda /opt/conda
+RUN useradd -rm -d /home/rapids -s /bin/bash -g root -u 1001 rapids
+
+COPY --from=condaforge/mambaforge:22.9.0-2 --chown=rapids /opt/conda /opt/conda
+
+USER rapids
+
+RUN \
+  # install expected Python version
+  mamba install -y -n base python="${PYTHON_VERSION}"; \
+  mamba update --all -y -n base; \
+  find /opt/conda -follow -type f -name '*.a' -delete; \
+  find /opt/conda -follow -type f -name '*.pyc' -delete; \
+  conda clean -afy;
+
+USER root
```

Comment:
Should we execute the apt-get and other "root" commands first, before copying in conda as the non-root user? Does the image need to end up in "root" mode?